docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
+
+## Intended audience
+
+This guide is intended for developers of InvenioRDM.
+
+## Overview
+
+Invenio administration panel gives the administrator a set of tools to effectively manage an instance of InvenioRDM. 
+
+## Domain dictionary
+
+The following terms are introduced to facilitate defining the domain:
+
+- `administration panel` - the interface panel enabling a `manager` to administrate the instance of InvenioRDM in a developer-independent way.
+- `administrator` - a person with domain knowledge, with a special set of permissions, able to manage an InvenioRDM instance, not necessarily having developer skills.
+- `frontsite`, `end user interface` - currently known InvenioRDM interface, accessible by anonymous and logged-in users, without administrator role.
+- `administration view` - subpages of the administration panel
+
+## Administration panel
+
+![Administration Panel](./img/administration/administration_wireframe.png)
+
+An instance developer can register new entries in the menu (see backend [RFC](https://github.com/inveniosoftware/rfcs/pull/67)). A menu entry will display the associated views in the panel section. The navbar menu layouts are customizable to meet the needs of the InvenioRDM instance.
```

Comment:
What is the `backend RFC`? Is it more the `back-office` or `administration` RFC?
Can we merge the RFC and link to the document and not the PR?

Reply:
yes, I am waiting for the review of the RFCs to be merged